.changeset/nice-comics-type.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/i18n": patch
```

Comment:
```suggestion
"@rocket.chat/meteor": minor
"@rocket.chat/i18n": minor
```
(this should be marked as minor since this PR is not a fix)

Reply:
Thanks I did not pay attention to this. Updated